test/AWS_Tests/src/Inter_Backend_File_Operations_Spec.enso

```diff
@@ -0,0 +1,158 @@
+## This test file checks operations on files that are happening between various backends.
+
+   Because it relies not only on Standard.Base but also the S3 backend provided
+   by Standard.AWS, it is currently placed in `AWS_Tests`.
+   Once we start supporting more backends, we should consider creating
+   a separate test project for these integrations (e.g. `Integrator_Tests`).
```

Comment:
I think we may want to move this to a separate test project, but for now seems enough to keep in AWS.